Rakefile

```diff
@@ -1,14 +1,13 @@
 require "bundler/setup"
 require "bundler/gem_tasks"
 require "bump/tasks"
-require "wwtd/tasks"
 require "appraisal"
```

Comment:
Do you still use Appraisal, @grosser, or do you edit gemfiles manually? (I've edited them manually in this PR)